tests/test_optimizers.py

```diff
@@ -49,12 +49,19 @@ def run_and_compare(self, output_names_with_port, onnx_feed_dict, origin_proto,
             self.assertEqual(expected_val.dtype, actual_val.dtype)
             self.assertEqual(expected_val.shape, actual_val.shape)
 
+        return new_proto
     # Tranpose Optimizer Tests Start
 
     def run_transpose_compare(self, output_names_with_port, onnx_feed_dict, origin_proto,
                               remaining_transpose_num=None, debug=False, rtol=1e-07):
-        self.run_and_compare(output_names_with_port, onnx_feed_dict, origin_proto, op_type="Transpose",
-                             remaining_op_num=remaining_transpose_num, debug=debug, rtol=rtol)
+        return self.run_and_compare(output_names_with_port, onnx_feed_dict, origin_proto, op_type="Transpose",
+                                    remaining_op_num=remaining_transpose_num, debug=debug, rtol=rtol)
+
+    def check_transpose_perm(self, model_proto, expected_perm):
```

Comment:
Optional: take it as model_checker option of run_transpose_compare like graph_checker in backend_test_base.py?